src/sentry/api/helpers/group_index/update.py

```diff
@@ -299,110 +299,133 @@ def handle_resolve_in_release(
     user: User | RpcUser,
     result: MutableMapping[str, Any],
 ) -> tuple[dict[str, Any], GroupResolution.Type | None] | Response:
-    res_type = None
-    release = None
-    commit = None
     self_assign_issue = "0"
     if acting_user:
         user_options = user_option_service.get_many(
             filter={"user_ids": [acting_user.id], "keys": ["self_assign_issue"]}
         )
         if user_options:
             self_assign_issue = user_options[0].value
-    res_status = None
-    if status == "resolvedInNextRelease" or status_details.get("inNextRelease"):
+
+    res_type_str = "now"
+    activity_type = ActivityType.SET_RESOLVED.value
+    activity_data = {}
+    new_status_details = {}
```

Comment:
This is the default behaviour:
https://github.com/getsentry/sentry/blob/7efb91bbb24f72bc0b1db7356ad89bef4d8016a3/src/sentry/api/helpers/group_index/update.py#L399-L403